modules/space/rendering/renderableorbitalkepler.cpp

```diff
@@ -173,9 +305,29 @@ RenderableOrbitalKepler::RenderableOrbitalKepler(const ghoul::Dictionary& dict)
     _segmentQuality.onChange([this]() { updateBuffers(); });
     addProperty(_segmentQuality);
 
-    _appearance.lineColor = p.color;
-    _appearance.lineFade = p.trailFade.value_or(20.f);
-    _appearance.lineWidth = p.lineWidth.value_or(2.f);
+    _appearance.color = p.color;
+    _appearance.trailFade = p.trailFade.value_or(_appearance.trailFade);
+    _appearance.trailWidth = p.trailWidth.value_or(2.f);
```

Comment:
```suggestion
    _appearance.trailWidth = p.trailWidth.value_or(_appearance.trailWidth );
```

Otherwise we might have a wrong default value if we change the default value of the property above